### PR TITLE
Fix webpack 3 async imports

### PR DESCRIPTION
**Pull request by marcodejongh**
This was throwing exceptions in the browser. Apparently the dynamic import behaviour of files exporting a module throwing "module.exports" has changed between webpack 3 and 4

### CODE REVIEW

**Comment by conorhastings** [comment]
looks like test is failing, seeing array of items passed to className?

**Comment by marcodejongh** [comment]
@conorhastings  Should go green now. 

I've also been working on a version that async loads the languages, just been slow going

File: src/async-syntax-highlighter.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ export default (options) => {
     static languages = [];
 
     static preload() {
-      ReactAsyncHighlighter.loadAstGenerator();
+      return ReactAsyncHighlighter.loadAstGenerator();
```
Comment (marcodejongh) [inline src/async-syntax-highlighter.js]: The snapshots on master actually don't have syntax highlighting. This was caused by not returning the astGenerator promise in the preload method

**Comment by conorhastings** [comment]
looks good, i'll publish a new version tonight when i'm home from work

**Pull request merged**